mmspheatpump.php

```diff
@@ -184,7 +184,7 @@
 
                 <div class="section-heading">Pumps</div>
 
-                <div data-config-keys="WaterPump1Status,WaterPump2Status,WaterPump4Status" class="chart"></div>
+                <div data-config-keys="OperationMode,WaterPump1Status,WaterPump2Status,WaterPump4Status" class="chart"></div>
```

Comment:
I was worried about the gold, but it makes sense using this here. It'll compensate for the WaterPumpXStatus having gone AWOL on my feeds.